src/recordpage.cpp

```diff
@@ -207,24 +207,30 @@ RecordPage::RecordPage(QWidget* parent)
     ui->outputFile->setMode(KFile::File | KFile::LocalOnly);
     ui->eventTypeBox->lineEdit()->setPlaceholderText(tr("perf defaults (usually cycles:Pu)"));
 
-    ui->launchConfigComboBox->setEditable(true);
-    ui->launchConfigComboBox->setDisabled(true);
-    ui->launchConfigComboBox->setInsertPolicy(QComboBox::InsertAtCurrent);
-    connect(ui->launchConfigComboBox->lineEdit(), &QLineEdit::editingFinished, this,
-            &RecordPage::onApplicationConfigRenamed);
-    connect(ui->launchConfigComboBox, qOverload<int>(&QComboBox::currentIndexChanged), this,
-            &RecordPage::onApplicationConfigChanged);
-    connect(ui->addConfig, &QPushButton::clicked, this, &RecordPage::onApplicationConfigAdded);
-    connect(ui->removeConfig, &QPushButton::clicked, this, &RecordPage::onApplicationConfigRemoved);
-
-    const auto updateLaunchConfig = [this] {
-        if (ui->launchConfigComboBox->count() > 0) {
-            saveApplicationConfig(ui->launchConfigComboBox->currentText());
-        }
+    m_multiConfig = new MultiConfigWidget(this);
+
+    m_multiConfig->setKConfigGroup(applicationConfig(ui->applicationName->text()));
+
+    auto saveFunction = [this](KConfigGroup group) {
+        group.writeEntry(QLatin1String("params"), ui->applicationParametersBox->text());
```

Comment:
why the QL1S here now suddenly? shouldn't be needed, and isn't used for `readEntry` below either?